docs/ADRs/0005-python-tooling.md

```diff
@@ -0,0 +1,44 @@
+---
+    title: 0005 Python coding standard and tools
+    adr:
+        author: Berry den Hartog
+        created: 28-Feb-2024
+        status:  accepted
+---
+
+## Context
+
+In modern software development, maintaining code quality is crucial for readability, maintainability, and collaboration.
+Python, being a dynamically typed language, requires robust tooling to ensure code consistency and type safety. Manual
+enforcement of coding standards is time-consuming and error-prone. Hence, adopting automated tooling to streamline this
+process is imperative.
+
+## Decision
+
+We will use these standards and tools for our own projects:
+
+- [google style guide](https://google.github.io/styleguide/pyguide.html)
+- [Black](https://black.readthedocs.io/en/stable/): A code formatter that enforces a consistent code style throughout the
+project.
+- [Pyright](https://microsoft.github.io/pyright/#/): A static type checker for Python, ensuring type safety and reducing
```

Comment:
Why pyright over mypi?

Reply:
is veel sneller

Reply:
pylint is not a static type checking tool though, is it?

Reply:
Not as far is a know. it is a linting tool like flake8

Reply:
ah sorry that is my misunderstanding then :s